src/sst/core/testingframework/sst_unittest_support.py

```diff
@@ -127,19 +127,13 @@ def testing_is_PIN_loaded() -> bool:
     pin_path = os.environ.get('INTEL_PIN_DIRECTORY')
     if pin_path is not None:
         pindir_found = os.path.isdir(pin_path)
-    #log_debug("testing_is_PIN_loaded() - Intel_PIN_Path = {0}; Valid Dir = {1}".format(pin_path, pindir_found))
     return pindir_found
 
 def testing_is_PIN_Compiled() -> bool:
     global pin_exec_path
     pin_crt = sst_elements_config_include_file_get_value(define="HAVE_PINCRT", type=int, default=0, disable_warning=True)
     pin_exec = sst_elements_config_include_file_get_value(define="PINTOOL_EXECUTABLE", type=str, default="", disable_warning=True)
-    #log_debug("testing_is_PIN_Compiled() - Detected PIN_CRT = {0}".format(pin_crt))
-    #log_debug("testing_is_PIN_Compiled() - Detected PIN_EXEC = {0}".format(pin_exec))
-    pin_exec_path = pin_exec
```

Comment:
Not sure if it's needed, but I noticed that the side effect of setting the global variable pin_exec_path is not in the new code.  Was this intended?

Reply:
This wasn't intended, so I either need to put it back or check in all of the core and elements Python code for its usage.